creator-node/src/logging.js

```diff
@@ -63,4 +63,59 @@ function loggingMiddleware (req, res, next) {
   next()
 }
 
-module.exports = { logger, loggingMiddleware, requestNotExcludedFromLogging, getRequestLoggingContext }
+/**
+ * Add fields to a child logger instance
+ * @param {*} req
+ * @param {Object} options fields to add to child logger
+ * @returns a logger instance
+ */
+function setFieldsInChildLogger (req, options = {}) {
+  const fields = Object.keys(options)
+
+  const childOptions = {}
+  fields.forEach(field => {
+    childOptions[field] = options[field]
+  })
+
+  return req.logger.child(childOptions)
+}
+
+/**
+ * Pulls the start time of the req object to calculate the duration of the fn
+ * @param {*} req
+ * @returns the duration of the fn call in ms
+ */
+function getDuration (req) {
+  let durationMs
+  if (req.startTime) {
+    const endTime = process.hrtime(req.startTime)
+    durationMs = Math.round(endTime[0] * 1e3 + endTime[1] * 1e-6)
+  }
+
+  return durationMs
+}
+
+/**
+ * Prints the log message with the duration
+ * @param {*} req
+ * @param {string} msg the message to print
+ */
+function logInfoWithDuration (req, msg) {
+  const durationMs = getDuration(req)
+
+  if (durationMs) {
+    req.logger.info(`${msg} duration=${durationMs}`)
```

Comment:
Thoughts on turning this into a json field logged separately from the msg? 
so `req.logger.info({ duration }, msg)` - which would allow us to perform aggregate queries on the duration in the logging service.

Reply:
oh this is a good idea. trying

Reply:
omg this is perfect. changing it to your recommendation
```
[2021-12-13T22:34:22.919Z]  INFO: audius_creator_node/89738 on d6d084c2f9d0: Successfully served from fs! (requestID=Q-vmftHRqA, requestMethod=GET, requestHostname=34.69.50.225, duration=3)
    requestUrl: /ipfs/QmayFoV8cchJjSQAVidx7zAj55Qdpfr7vnZ8g4pP2D8fbQ
```